static-analysis/StaticAnalysisChapter.md

```diff
@@ -404,16 +403,15 @@ function loopcontents(e::Expr)
   end
   lines
 end
-~~~
+~~~~
 
 And now to explain in pieces:
 
-1. 
 ~~~.jl
 b = body(e)
 ~~~
 
-    We start by getting all the expressions in the body of method, as an `Array`. `body` is a function that I've already implemented:
+We start by getting all the expressions in the body of method, as an `Array`. `body` is a function that I've already implemented:
 
     ~~~.jl
```

Comment:
I'm not totally sure. Sometimes the indentation is important for formatting, like to make the paragraph part of the list element, but that's hard to determine other than trial-and-error with the markdown formatter.